src/disk_utils.cpp

```diff
@@ -642,9 +642,8 @@ int build_merged_vamana_index(std::string base_file, diskann::Metric compareMetr
         else
         {
             if (universal_label != "")
-            { //  indicates no universal label
-                LabelT unv_label_as_num = 0;
-                _index.set_universal_label(unv_label_as_num);
+            { //  indicates universal label
```

Comment:
universal_label is only use as bool?

Reply:
If a universal label exists, it will always be represented internally as the value 0. The existence of a universal label in a dataset is a boolean value.

Reply:
then we don't need to transfer std::string here? we can just use bool enable_universal_label?